CMakeLists.txt

```diff
@@ -268,6 +268,19 @@ if(WITH_BLUESTORE)
   endif()
 endif()
 
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "i386|i686|amd64|x86_64")
+  option(WITH_PMEM_DEVICE "Enable PMEM OSD" OFF)
+else()
+  # ARM64 PMDK support is still experimental
+  option(WITH_PMEM_DEVICE "Enable PMEM OSD" OFF)
```

Comment:
either way, it's disabled by default. why bother adding the `if`?

Reply:
I removed `if`.